api/v1/controllers/TeamController.js

```diff
@@ -2,28 +2,31 @@ var errors = require('../errors');
 var services = require('../services');
 var roles = require('../utils/roles');
 
+var bodyParser = require('body-parser');
+var middleware = require('../middleware');
 var router = require('express').Router();
 
 function joinTeam (req, res, next) {
   var email = req.auth.email;
-  var teamName = req.body['teamName'];
+  var teamName = req.body.teamName;
+
   return services.UserService.findUserByEmail(email)
   .then(function(user){
-    return services.TeamService
-      .addUserToTeam(user, teamName);
+	return services.TeamService
+	  .addUserToTeam(user, teamName);
   })
   .then(function(team){
-    return team.toJSON();
+	return team.toJSON();
```

Comment:
Yup. The team's toJSON function is a little different than the others we use. Check out the [Team.toJSON()](https://github.com/HackIllinois/api-2017/pull/22/files#diff-e183cda71d6c4948b0845c707463c6f0R33) function. It queries for the Users belonging to it and adds them to the JSON object. After that it returns a Promise containing the entire team JSON object.